contracts/CO30_Jonty_JackalsLoot.Woodland_ACR/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,4 @@
+<font color='#E3D310'>PRIMARY</font>
+Talk to Col. Artyom at the ambush location to get intel about stolen military assets.
+Once located, secure and prevent the assets from falling into Miyović's hands.
+Establish contact with CDF command and a recovery team will be dispatched to retrieve the assets.
```

Comment:
```suggestion
- Talk to Col. Artyom at the ambush location
- Secure the stolen military assets
- Request recovery team from CDF Command
```
Should be in point form and single action

In the mission you say
> Upon confirmation of the location, the support convoy will mobilize and provide additional firepower and manpower to secure the site.

Is it a recovery team coming to retrieve the assets, the support convoy coming to secure the site, both? You should clarify to prevent confusion